app/util/analytics.py

```diff
@@ -130,37 +124,46 @@ def get_actual_test_count(self):
             elif selenium_test in line:
                 self.selenium_test_count = self.selenium_test_count + 1
 
+    def __convert_to_sec(self, duration):
+        seconds_per_unit = {"s": 1, "m": 60, "h": 3600, "d": 86400, "w": 604800}
+        duration = str(duration)
+        numbers = ''.join(filter(str.isdigit, duration))
+        units = ''.join(filter(str.isalpha, duration))
+        return int(numbers) * seconds_per_unit[units] if units in seconds_per_unit else int(numbers)
+
     def generate_analytics(self):
         self.application_url = self.config_yml.server_url
-        self.run_id = self.id_generator(string=self.application_url)
         self.concurrency = self.config_yml.concurrency
-        self.duration = self.config_yml.duration
+        self.duration = self.__convert_to_sec(self.config_yml.duration)
         self.os = self.get_os()
         self.actual_duration = self.get_actual_run_time()
         self.tool_version = TOOLKIT_VERSION
         self.get_actual_test_count()
-        self.date = datetime.utcnow().strftime("%m/%d/%Y-%H:%M:%S")
+
 
 class AnalyticsSender:
 
     def __init__(self, analytics_instance):
-        self.run_analytics = analytics_instance
+        self.analytics = analytics_instance
 
     def send_request(self):
-        base_url = BASE_URL
-        params_string = f'date={self.run_analytics.date}&' \
-                        f'app_type={self.run_analytics.application_type}&' \
-                        f'os={self.run_analytics.os}&' \
-                        f'tool_ver={self.run_analytics.tool_version}&' \
-                        f'run_id={self.run_analytics.run_id}&' \
-                        f'exp_dur={self.run_analytics.duration}&' \
-                        f'act_dur={self.run_analytics.actual_duration}&' \
-                        f'sel_count={self.run_analytics.selenium_test_count}&' \
-                        f'jm_count={self.run_analytics.jmeter_test_count}&' \
-                        f'concurrency={self.run_analytics.concurrency}'
-
-        r = requests.get(url=f'{base_url}{params_string}')
-        if r.status_code != 403:
+        headers = {"Content-Type": "application/json"}
+        params = {"date": self.analytics.date,
+                  "time_stamp": datetime.now().timestamp(),
```

Comment:
This timestamp is float where digits before the coma are seconds. I think we do not need float numbers in our DB. Milliseconds will be a good choice so I suggest `int(round(datetime.utcnow().timestamp() * 1000))`

Also, I suggest don't do redundant operations and reuse variables:
``` 
       utc_now = datetime.utcnow()
        self.time_stamp = int(round(utc_now.timestamp() * 1000))
        self.date = utc_now.strftime("%m/%d/%Y-%H:%M:%S")
```